### PR TITLE
Add new requisition queries in requisition module

### PR DESCRIPTION
**Pull request by joshxg**
#831 - 1!

- This mainly adds the new queries and gets the pages working, though there is a lot more work to do!

### CODE REVIEW

File: packages/requisitions/src/SupplierRequisition/api/operations.graphql [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+# mutation deleteSupplierRequisitions($ids: [DeleteSupplierRequisitionInput!]) {
+#   batchSupplierRequisition(deleteSupplierRequisitions: $ids) {
+#     __typename
+#     deleteSupplierRequisitions {
+#       __typename
+#       id
+#     }
+#   }
+# }
```
Comment (joshxg) [inline packages/requisitions/src/SupplierRequisition/api/operations.graphql:33]: there's not batch mutation, so mass-deleting requisitions isn't ideal

File: packages/requisitions/src/SupplierRequisition/api/operations.graphql [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+  id
+  itemId
+  requestedQuantity
+  calculatedQuantity
+  itemStats {
+    averageMonthlyConsumption
+    stockOnHand
+    monthsOfStock
+  }
+}
```
Comment (joshxg) [inline packages/requisitions/src/SupplierRequisition/api/operations.graphql:45]: not sure about this, but not bad to gen the type from this subset of fields. Though, the type to use is `RequestRequisitionLineFragment` (and the same for the other fragments below) - I hate the suffix. Can use a config option to remove *all* suffixes, which would take a bit of time to rename the whole code base (i..e  `requisitionQuery` would be `requisition`). I also think fighting against this has always just ended up in more time spent, so thought to just go with it 🤷

File: packages/requisitions/src/utils.ts [hunk 3]
```diff
@@ -1,25 +1,25 @@
-import { SupplierRequisitionNodeStatus } from '@openmsupply-client/common';
+import { RequisitionNodeStatus } from '@openmsupply-client/common';
```
Comment (joshxg) [inline packages/requisitions/src/utils.ts:1]: Changes to get rendering to work, not exactly the right logic

**Comment by joshxg** [comment]
Have a list here of all the things to be done for request requisitions. Then repeat for response: https://github.com/openmsupply/openmsupply-client/issues/831

**Review by mark-prins — APPROVED** [review]
👍  looks good. maybe also rename the `SupplierRequisition` and `CustomerRequisition` folders? Is that too much?

**Comment by joshxg** [comment]
> Looks good. maybe also rename the SupplierRequisition and CustomerRequisition folders? Is that too much?

Will do - will just do a new PR to try make the diff easier to view. Unsure if the 'internal order' naming plays a part in the naming at all

**Pull request merged**